src/shared/createChartPurchaseByCompanies.js

```diff
@@ -0,0 +1,35 @@
+import Highcharts from 'highcharts';
+import Exporting from 'highcharts/modules/exporting';
+
+Exporting(Highcharts);
+
+const createChartPurchaseByCompanies = (data) => {
+  const series = [];
+  data.orders.forEach((company) => {
+    if (series.length === 0) return series.push(company);
+    const index = series.findIndex(serie => serie.companyId === company.companyId);
+    // eslint-disable-next-line no-return-assign
```

Comment:
Disabling eslint is the last resort.
Try to do this differently, so there is no need to disable it.